Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/viewmodel/SubmissionViewModel.kt

```diff
@@ -142,4 +166,52 @@ class SubmissionViewModel : ViewModel() {
             }
         }
     }
+
+    fun onNextClicked() {
+        symptomIntroductionEvent.value = SymptomIntroductionEvent.NavigateToSymptomCalendar
```

Comment:
Is there a special requirement that you directly assign the value, and not use the async `postValue` mechanism here?

Reply:
@Oliver-Zimmerman